HyphaApp/hypha
# Submission data on determination form page

Fixes #3971

## Test Steps

- Add or edit a determination (e.g. `/apply/submissions/2/determination/2/edit/`)
- See the proposal information on the right for added context
